rpcs3/Emu/RSX/D3D12/D3D12Buffer.cpp

```diff
@@ -147,8 +147,12 @@ std::tuple<std::vector<D3D12_SHADER_RESOURCE_VIEW_DESC>, size_t> upload_inlined_
 	// We can't rely on vertex_attribute_infos strides here so compute it
 	// assuming all attributes are packed
 	u32 stride = 0;
+	u32 initial_offsets[rsx::limits::vertex_count];
+	u8  index = 0;
```

Comment:
@kd-11 doesn't mean it's better/faster. At leas in any significant way. It'd be probably stored in register anyway.